libbeat/processors/decode_xml_fields/docs/decode_xml_fields.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[decode_xml_fields]]
```

Comment:
I think it would be nice to have small example showing the input/output of this processor.